api/v1beta1/spannerautoscaler_types.go

```diff
@@ -108,6 +108,14 @@ type ScaleConfig struct {
 	// The cool down period between two consecutive scaledown operations. If this option is omitted, the value of the `--scale-down-interval` command line option is taken as the default value.
 	ScaledownInterval *metav1.Duration `json:"scaledownInterval,omitempty"`
 
+	// The maximum number of processing units which can be added in one scale-up operation. It can be a multiple of 100 for values < 1000, or a multiple of 1000 otherwise.
+	// +kubebuilder:default=2000
+	ScaleupStepSize int `json:"scaleupStepSize,omitempty"`
+
+	// How often autoscaler is reevaluated for scale up.
+	// The warm up period between two consecutive scaleup operations. If this option is omitted, the value of the `--scale-up-interval` command line option is taken as the default value.
+	ScaleupInterval *metav1.Duration `json:"scaleupInterval,omitempty"`
```

Comment:
Same as ScaleupStepSize

Reply:
I fixed it to 10 seconds because default interval for scaleup was set to 10 seconds before introducing scaleup feature.
https://github.com/mercari/spanner-autoscaler/pull/132/files#diff-1984e42ef0a0d5e7edf140b3ae6decf3fd6e1b9bd4ee351d4dcd8cf7213bb537L488